### PR TITLE
Fix vulnerabilities

### PR DESCRIPTION
**Pull request by sbckr**

### CODE REVIEW

**Comment by sbckr** [comment]
cannot tell reasons for the codecov check to fail, but its own report sais coverage has increased:

> # [Codecov](https://codecov.io/gh/carbynestack/amphora/pull/44?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=carbynestack) Report
> 
> > Merging [#44](https://codecov.io/gh/carbynestack/amphora/pull/44?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=carbynestack) ([0159659](https://github.com/carbynestack/amphora/commit/015965900d7ab52fee0ee71a028ac85d4d425cf4)) into [master](https://codecov.io/gh/carbynestack/amphora/commit/2fa009a8b689cc2345a39ed6bbde268d6e9b4f09?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=carbynestack) ([2fa009a](https://github.com/carbynestack/amphora/commit/2fa009a8b689cc2345a39ed6bbde268d6e9b4f09)) will **increase** coverage by `51.22%`.

**Comment by sbckr** [comment]
E2E tests successful

**Comment by sbckr** [comment]
Blocked by carbynestack/castor#48 to upgrade castor version

File: amphora-common/src/test/java/io/carbynestack/amphora/common/entities/SecretShareTest.java [hunk 1]
```diff
@@ -38,7 +36,7 @@ public void givenDataOfInvalidLength_whenBuildSecretShare_thenThrowIllegalArgume
   }
 
   @Test
-  public void givenTwoSecretShareWithSameContent_whenCompareEqual_thenMatch() {
+  void givenTwoSecretShareWithSameContent_whenCompareEqual_thenMatch() {
```
Comment (strieflin) [inline amphora-common/src/test/java/io/carbynestack/amphora/common/entities/SecretShareTest.java:39]: Typo: Share**s**

File: amphora-java-client/src/test/java/io/carbynestack/amphora/client/DefaultAmphoraClientTest.java [hunk 2]
```diff
@@ -250,16 +246,13 @@ public void givenUploadingMaskedInputFails_whenCreateObject_thenThrowAmphoraClie
...
-            String.format(REQUEST_FOR_ENDPOINT_FAILED_EXCEPTION_MSG, testUri, 500)));
+    assertThat(actualAce.getMessage())
+        .endsWith(String.format(REQUEST_FOR_ENDPOINT_FAILED_EXCEPTION_MSG, testUri, 500));
   }
 
   @SneakyThrows
   @Test
-  public void
-      givenOutPutDeliveryObjectsOfVariousSize_whenDownloadingObject_thenReturnExpectedContent() {
+  void givenOutPutDeliveryObjectsOfVariousSize_whenDownloadingObject_thenReturnExpectedContent() {
```
Comment (strieflin) [inline amphora-java-client/src/test/java/io/carbynestack/amphora/client/DefaultAmphoraClientTest.java:255]: Typo: Out**p**ut

File: amphora-java-client/src/test/java/io/carbynestack/amphora/client/DefaultAmphoraInterVcpClientTest.java [hunk 3]
```diff
@@ -45,14 +44,14 @@ public class DefaultAmphoraInterVcpClientTest {
...
-  @Before
+  @BeforeEach
   public void setUp() {
     this.amphoraInterVcpClient =
         new DefaultAmphoraInterVcpClient(amphoraServiceUris, amphoraCommunicationClient);
   }
 
   @Test
-  public void givenNoServiceUriDefined_whenBuildingInterVcClient_thenThrowException() {
+  void givenNoServiceUriDefined_whenBuildingInterVcClient_thenThrowException() {
```
Comment (strieflin) [inline amphora-java-client/src/test/java/io/carbynestack/amphora/client/DefaultAmphoraInterVcpClientTest.java:54]: Type: InterVc*p*Client

File: amphora-service/src/test/resources/application-test.properties [hunk 4]
```diff
@@ -0,0 +1,29 @@
+spring.config.activate.on-profile=test
```
Comment (strieflin) [inline amphora-service/src/test/resources/application-test.properties:1]: Missing license header. Please also check for rest of codebase.

**Review by strieflin — CHANGES_REQUESTED** [review]
Found some smaller issues, only (see other comments).

**Review by strieflin — APPROVED** [review]
LGTM

**Pull request merged**